llama-index-core/llama_index/core/workflow/utils.py

```diff
@@ -53,6 +53,9 @@ def validate_step_signature(fn: Callable) -> Tuple[str, List[object], List[objec
         if t.annotation == Context:
             continue
 
+        if str(t.annotation) == "<class 'llama_index.core.workflow.workflow.Workflow'>":
```

Comment:
Maybe this for loop should be inversed -- only looking for events

Reply:
yes we shouldn't merge it, just a dirty trick to have a working example 😅

Reply:
I rewrote the validation strategy, `validate_step_signature` is not used anymore